slices/fonts-noto-core.yaml

```diff
@@ -0,0 +1,210 @@
+package: fonts-noto-core
+
+essential:
+  - fonts-noto-core_copyright
+
+slices:
+  fonts:
+    essential:
+      - fonts-noto-mono_fonts
```

Comment:
Shouldn't the `fonts-noto-core_config` slice be listed as a dependency here?

Reply:
Any thoughts?

_(Was this previously requested to be secluded?)_

Reply:
@rebornplusplus not really — I can do core slice which will contain both of them, but in general @cjdcordeiro ask me to split packages to fonts and configs which aren't dependant on each other.